_includes/js/repeater.html

```diff
@@ -262,6 +262,12 @@ <h3 id="repeater-usage-options">Options</h3>
           the rendering. Review more details on using <a href="#repeater-dataSource">dataSource</a> below.</td>
       </tr>
       <tr>
+        <td>dataSourceOptions</td>
+        <td>object</td>
+        <td>null</td>
+        <td>Use this object to pass a parameter to a custom-defined dataSource function mentioned above. Then, use those values to customize the data that gets passed back into the repeater from your API. Suggested uses include sorting, filtering, and search values. This object is also valuable in custom renders when used as an object within the <code>render</code> methods's <a href="#repeater-usage-methods-render">options parameter</a>. Did you lose your custom <code>dataSourceOptions</code> when you changed the page? If you did, set <code>preserveDataSourceOptions</code> to true in order to keep them.</td>
```

Comment:
"Did you lose your custom <code>dataSourceOptions</code> when you changed the page?"

I would change this to "Did you lose your custom <code>dataSourceOptions</code> when you switched pages in the repeater?" just to clarify that we're not referring to changing the browser page.